pkgs/tools/filesystems/gcsfuse/default.nix

```diff
@@ -0,0 +1,23 @@
+# This file was generated by go2nix.
+{ stdenv, buildGoPackage, fetchgit, fetchhg, fetchbzr, fetchsvn }:
+
+buildGoPackage rec {
+  name = "gcsfuse-${version}";
+  version = "81281027";
```

Comment:
If there is no release, use the following format: `<year>-<month>-<day>` instead of the git commit.